programs/lfx-mentorship/2024/02-Jun-Aug/README.md

```diff
@@ -693,4 +693,4 @@ Mentee application instructions can be found on the [Program Guidelines](https:/
     - Michael Yuan (@juntao, michael@secondstate.io)
     - Hung-Ying Tai (@hydai, hydai@secondstate.io)
 - Upstream Issue: https://github.com/WasmEdge/WasmEdge/issues/3371
-- LFX URL: https://mentorship.lfx.linuxfoundation.org/project/d52d172e-598d-4817-be97-3338d5b96432
+- LFX URL: https://mentorship.lfx.linuxfoundation.org/project/d52d172e-598d-4817-be97-3338d5b96432
```

Comment:
I'm not sure what this change is...

Reply:
Probabaly a space change, thanks for pointing out, lemme remove this

Reply:
thanks :)

Reply:
Fixed